sycl/include/CL/sycl/detail/pi.hpp

```diff
@@ -331,6 +334,7 @@ class DeviceBinaryImage {
   pi_device_binary Bin;
   pi::PiDeviceBinaryType Format = PI_DEVICE_BINARY_TYPE_NONE;
   DeviceBinaryImage::PropertyRange SpecConstIDMap;
+  DeviceBinaryImage::PropertyRange SpecConstDefaultValuesMap;
```

Comment:
It looks like a change which break backward compatibility since this class crosses library boundary.
Suggest using `getProperty` method instead.